game/scripts/npc/abilities/lich_dark_ritual.txt

```diff
@@ -10,7 +10,7 @@
 		//-------------------------------------------------------------------------------------------------------------
 		"ID"					"5136"														// unique ID number for this ability.  Do not change this once established or it will invalidate collected stats.
 		"AbilityBehavior"				"DOTA_ABILITY_BEHAVIOR_UNIT_TARGET | DOTA_ABILITY_BEHAVIOR_DONT_RESUME_ATTACK"
-		"AbilityUnitTargetTeam"			"DOTA_UNIT_TARGET_TEAM_FRIENDLY"
+		"AbilityUnitTargetTeam"			"DOTA_UNIT_TARGET_TEAM_ENEMY | DOTA_UNIT_TARGET_TEAM_FRIENDLY"
```

Comment:
"AbilityUnitTargetTeam"			"DOTA_UNIT_TARGET_TEAM_BOTH" - You can do this instead.